contracts/core/Core.sol

```diff
@@ -19,6 +19,7 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "./CoreI.sol";
 import "./CoreStatusEnum.sol";
 import "../consensus/ConsensusModule.sol";
+import "../validator-set/ValidatorSet.sol";
```

Comment:
Since we have touched this section, can we sort all imports as per the dev guidelines?

Reply:
Done.